docs/reference/action_list.rst

```diff
@@ -801,3 +841,77 @@ Combine this with configuring a custom template and you'll have a list column fu
 
 .. _`SonataDoctrineORMAdminBundle Documentation`: https://docs.sonata-project.org/projects/SonataDoctrineORMAdminBundle/en/4.x/reference/list_field_definition/
 .. _`here`: https://github.com/sonata-project/form-extensions/tree/1.x/src/Type
+
+Advance Usage
+-------------
+
+Displaying sub entity properties
```

Comment:
Since this is `SonataAdminBundle` I would try to use `model` instead of `entity` in general.

Reply:
Done